.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI Build
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  ci_build:
+    name: CI Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install x86_64-pc-windows-gnu
```

Comment:
Instead of going through the release steps on the push/pull_request triggers, you can just run `cargo test` to run the test suite.